packages/bff/src/model/domain/apiConverter.ts

```diff
@@ -107,15 +107,12 @@ export const toApiAttributeProcessSeed = (
 });
 
 export function toAuthorizationKeySeed(
-  seed: bffApi.KeySeed,
-  userId: string
+  seed: bffApi.KeySeed
 ): authorizationApi.KeySeed {
   return {
-    userId,
     key: seed.key,
     use: seed.use,
     alg: seed.alg,
     name: seed.name,
-    createdAt: new Date().toISOString(),
   };
 }
```

Comment:
@paologaleotti could you please check why you added `userId` and `createdAt`? it seems they are not present in the model.